File: NativePlugins/Bullet Holic Physics/Bullet Holic Physics/Bullet.cpp
#include "pch.h"
#include "Bullet.h"




	Bullet::~Bullet() {}

	Bullet::Bullet(int ID) {

		id = ID;
	}
	Bullet::Bullet() {
	
		id = 0;
	}
	
	Bullet::Bullet(Vector2 Position, Vector2 Velocity, Vector2 Acceleration, int Id) {

		position = Position;
		velocity = Velocity;
		acceleration = Acceleration;
		id = Id;


	}

	Bullet Bullet::ReturnBullet(){

		return *this;


	}

File: Assets/Scripts/Bullet.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using System.Runtime.InteropServices;



public class Bullet : MonoBehaviour
{

    SpriteRenderer currSprite; 

    float damage;
    public int id;

    public void AddBulletData(Vector2 Pos, Sprite sprite, float Damage, int ID)
    {
        transform.position = Pos;
        currSprite.sprite = sprite;
        damage = Damage;
        id = ID; 
        
    }
    
    void MoveBullet(Vector2 newPos)
    {
        transform.position = newPos;

    }
   public void UpdateBullet(Vector2 Pos) { 

        


    }
}